open_bounties/0079 Python Code Samples 2.md

```diff
@@ -0,0 +1,39 @@
+---
+ID: 0079
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/79
+Category: Technical Documentation
+Status: Open
+Amount: $5500
+---
+
+# Python Code Samples 2
+
+## Overview
+
+Currently, our Python documentations have [code snippets](https://github.com/XRPLF/xrpl-py/tree/master/snippets) and [code samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples) for some common usecases. However, there needs to be more examples to capture common tasks that users would want the library to do. Developers would benefit from additional relevant, high quality, well-documented examples in Python.
+
+## Details
+
+To add missing code samples, follow the instructions at the bottom of the page, under **Contribute Code Samples**.
+
+https://xrpl.org/code-samples.html
+
+## Milestones
+
+| #   | Description              | Details                                                                                                                                                                                                    | Proposed Potential Award | Open? |
+| --- | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ | ----- |
+| 1   | NFT                      | Mint (or batch mint) and burn an [NFT Token](https://xrpl.org/non-fungible-tokens.html#non-fungible-tokens-overview), assign an authorized minter, transfer NFT tokens (create, cancel, and accept offer). | $1000                    | Open  |
+| 2   | Require Destination Tags | Enable [Require Destionation Tags](https://xrpl.org/require-destination-tags.html#require-destination-tags) for an account.                                                                                | $500                     | Open  |
```

Comment:
Since this is for xrpl.org only we can remove Use Tickets, Escrow History, and Tickets should be removed.

Reply:
I don't think Tickets exists on xrpl.org for Python (List which [Tickets](https://xrpl.org/use-tickets.html) are outstanding against one’s own account and use Tickets to collect signatures for multisign transactions)

Reply:
use tickets does

Reply:
I didn't realize this bounty was for updating the code in guides.  Do we need to update the description of the bounty then?

Reply:
https://xrpl.org/code-samples.html

Reply:
@jonathanlei This is where I was looking. https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/_code-samples/use-tickets/py/use-tickets.py

Reply:
yeah description could be more clear

Reply:
ahh then we should just port it over instead of having it as a bounty

Reply:
unless it's not updated

Reply:
yeah but that's for use-tickets, Tickets have a different description - List which [Tickets](https://xrpl.org/use-tickets.html) are outstanding against one’s own account and use Tickets to collect signatures for multisign transactions.